DAO.sol

```diff
@@ -821,7 +821,8 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
+			&& lastTimeMinQuorumMet < (now + minProposalDebatePeriod)) {
```

Comment:
this protect the DAO from a malicous Curator, he calls this function again and again, to set it close to 0. Now the min time between calling 
'halveMinQuorum` is 2 weeks. When the DAO sees the attack from the curator, they should split.